### ISSUE TITLE
Magazine-specific search

### ISSUE THREAD

**Issue by puppechow** [hariettem/Artemis#260]
**Is your feature request related to a problem? Please describe.**
Whenever I enter an informational community, I usually try to perform a search for some key words to see if my question/discussion has already been posted before I make a post myself. It seems that in kbin and the Artemis app, you can only search for magazines.

**Describe the solution you'd like**
An accessible button to allow a search from within the active magazine would be nice!

**Describe alternatives you've considered**
N/A

**Additional context**
My only concern with this request is that it may clutter the UI depending on its placement.

**Comment by hyplex1** [comment]
+1 for search functionality. The ability to search for post titles or key-words in comments would be helpful. e.g. the fix for the text/link overlap issue was resolved and I was trying to find the same post that I called out in issue 259 to validate and I had trouble finding both the malicious compliance mag and the specific post from within Artemis.

**Comment by RheingoldRiver** [comment]
I do not believe this is possible upstream yet, so this would be waiting for implementation on /kbin's end.
